spi/build.gradle

```diff
@@ -124,9 +124,8 @@ publishing {
               }
               developers {
                 developer {
-                  id = "amazonwebservices"
-                  organization = "Amazon Web Services"
-                  organizationUrl = "https://aws.amazon.com"
+                  name = "OpenSearch"
+                  url = "https://github.com/opensearch-project/OpenSearch"
```

Comment:
Another one, I promise its the last :)

Reply:
:) just saw that too thats where this was copied from.  Updated!